src/layouts/base-new/index.tsx

```diff
@@ -95,8 +84,8 @@ const BaseNewLayout = ({
 					hideOnMobile
 				/>
 			)}
-			<Layout {...layoutProps}>
-				<div className={s.root} data-layout="base-new">
+			<CoreDevDotLayoutWithTheme theme={theme}>
+				<div className={s.root} data-CoreDevDotLayoutWithTheme="base-new">
```

Comment:
⛏️ What does `data-layout` (or now `data-CoreDevDotLayoutWithTheme`) do? I took a peek in code and couldn't find anything... (searched for both `data-layout` and `dataset` use, and found neither).

I have a feeling we _might_ be able to drop this 🤔 

```suggestion
				<div className={s.root}>
```

Reply:
Oh whoops I think that was a find and replace error from me. I'll look into data-layout though to validate whether we still need that. Thanks for the catch!

Reply:
I updated it back to the original but im a bit unsure of removing in this PR since I don't have the context on what it used to be used for!